scipy/stats/_continuous_distns.py

```diff
@@ -3717,6 +3850,8 @@ class rice_gen(rv_continuous):
 
     `rice` takes ``b`` as a shape parameter.
 
+    %(after_notes)s
+
```

Comment:
Honestly, I don't think we can change rice itself. Backwards compat is one reason, a deeper one is I'm not sure it's too deep of a rabbit hole to start special-casing the behavior of individual distributions with respect to shifting/scaling. (This equally applies to `expon` @josef-pkt mentiones above in this thread: technically it's easy to rename scale to lambda, but FWIW I don't think we should do it). 

So overall I'd view this as a documentation issue. Spefically, how do you see this comment in light of https://github.com/scipy/scipy/pull/4611 -- do you think we should have both @efiring ?